recipes-bsp/u-boot/u-boot-remarkable2_2018.03.bb

```diff
@@ -0,0 +1,33 @@
+DESCRIPTION = "U-Boot for reMarkble2"
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://Licenses/README;md5=a2c678cfd4a4d97135585cad908541c6"
+
+require recipes-bsp/u-boot/u-boot.inc
+
+PV = "2018.03+git${SRCPV}"
+
+SRCREV = "47c91918aa7724c16b6eaf87cf5dfbf4548eafc1"
+SRCBRANCH = "zero-sugar"
+SRC_URI = "git://github.com/reMarkable/uboot.git;branch=${SRCBRANCH} \
```

Comment:
Have you checked if the U-Boot mainline offers support for it? If so, the patches could be upstreamed and we might use u-boot-fslc.

Reply:
I haven't checked yet. I'm currently working on upstreaming Linux, after that I can look at u-boot. They use u-boot to do some customised setup so I don't think mainline u-boot would work out of the box.

Reply:
Ok; we could apply the patches to linux-fslc in this case.

Reply:
Do you mean `u-boot-fscl`?

There are more then a few patches, I think that will be difficult to do. My hope is just to upstream the diff and then use mainline. I'm hoping to be able to swap to a mainline kernel + patches soon-ish.